pipelines/cdp/scripts/01_create_data_sources.sh

```diff
@@ -0,0 +1,59 @@
+
+bq --location=us-central1 mk -d \
```

Comment:
Remove hardcoded regions from the script, and use an environment variable.

The 00 script should be generated by Terraform, and should populate an env variable, that is later used in this (and other) script.

Reply:
Actually, same as you are already doing below with PROJECT, BQ_CDP_DATASET, etc